modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/preloader/GridDhtPartitionsExchangeFuture.java

```diff
@@ -308,6 +308,15 @@
     @GridToStringExclude
     private volatile IgniteDhtPartitionHistorySuppliersMap partHistSuppliers = new IgniteDhtPartitionHistorySuppliersMap();
 
+    /** Set of nodes that cannot be used for wal rebalancing due to some reason. */
+    private Set<UUID> exclusionsFromWalRebalance = Collections.newSetFromMap(new ConcurrentHashMap<>());
```

Comment:
I suggest to rename it ot exclusionsFromWALRebalance or exclusionsFromHistoricalRebalance.

Reply:
Renamed to `exclusionsFromHistoricalRebalance`